readme.txt

```diff
@@ -98,6 +98,12 @@ Choose from one of 20 pre-defined color palettes or use your own custom color sc
 = Where can I find documentation and learn how to use the plugin? =
 We have tutorials, videos and other helpful information on the [Draw Attention website](https://wpdrawattention.com/?utm_source=website&utm_medium=promo-link&utm_campaign=wordpress-org&utm_content=docs-question)
 
+= I have a question or need help with the plugin, how do I contact your support team? =
+
+To contact our support team, send us an email through the `License & Support` tab within the plugin. Or, submit your message thoruhg our website's [Contact page](https://wpdrawattention.com/contact/).
```

Comment:
@gnarza Draw Attention Free doesn't have a `License & Support` tab. Maybe we should add a tab for support to the free version in the future, for now let's update this text to suggest that they contact us through the wp.org support forums or through our website and our pro support team will respond.

![Screen Shot 2023-05-15 at 2 24 55 PM](https://github.com/nsquared-team/draw-attention/assets/232096/d569fe14-88c8-4b6a-88ee-8cd367b51a2e)